[2022-11-04 12:21:14,481] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-11-04T12:21:13.527934+00:00 [queued]>
[2022-11-04 12:21:14,491] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-11-04T12:21:13.527934+00:00 [queued]>
[2022-11-04 12:21:14,492] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-04 12:21:14,494] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-11-04 12:21:14,496] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-04 12:21:14,508] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): t1> on 2022-11-04 12:21:13.527934+00:00
[2022-11-04 12:21:14,515] {standard_task_runner.py:52} INFO - Started process 2568 to run task
[2022-11-04 12:21:14,518] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xcom_dag', 't1', 'manual__2022-11-04T12:21:13.527934+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpox3j32ai', '--error-file', '/tmp/tmpfx0rbp21']
[2022-11-04 12:21:14,533] {standard_task_runner.py:80} INFO - Job 44: Subtask t1
[2022-11-04 12:21:14,611] {task_command.py:369} INFO - Running <TaskInstance: xcom_dag.t1 manual__2022-11-04T12:21:13.527934+00:00 [running]> on host 0636e12a55b7
[2022-11-04 12:21:14,682] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-11-04T12:21:13.527934+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-04T12:21:13.527934+00:00
[2022-11-04 12:21:14,697] {python.py:173} INFO - Done. Returned value was: None
[2022-11-04 12:21:14,707] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=t1, execution_date=20221104T122113, start_date=20221104T122114, end_date=20221104T122114
[2022-11-04 12:21:14,754] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-04 12:21:14,792] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
